windows-release/test-msi.ps1

```diff
@@ -0,0 +1,100 @@
+param ([string]$SetupExe)
+
+Write-Host "##[section]Install Python"
+$SetupCmd = "$SetupExe " + `
+            "/passive /log ""C:\Logs\install\log.txt"" " + `
+            "TargetDir=C:\Python " + `
+            "Include_debug=1 " + `
+            "Include_symbols=1 " + `
+            "InstallAllUsers=${env:InstallAllUsers} " + `
+            "${env:IncludeFreethreadedOpt}"
+Write-Host "##[command]$SetupCmd"
+iex $SetupCmd
+if ($LASTEXITCODE) { exit $LASTEXITCODE }
+
+Write-Host "##[command]dir C:\Python"
+dir C:\Python
+
+$env:PATH = "C:\Python:${env:PATH}"
+
+
+Write-Host "##[section]Capture Start Menu items"
+Write-Host "##[command]dir -r ""${env:PROGRAMDATA}\Microsoft\Windows\Start Menu\Programs\Python*"""
+dir -r "${env:PROGRAMDATA}\Microsoft\Windows\Start Menu\Programs\Python*"
+
+Write-Host "##[command]dir -r ""${env:APPDATA}\Microsoft\Windows\Start Menu\Programs\Python*"""
+dir -r "${env:APPDATA}\Microsoft\Windows\Start Menu\Programs\Python*"
+
+Write-Host "##[section]Capture registry"
+Write-Host 'Capture per-machine 32-bit registry'
+# PS 5.0 and later can't do this, because they normalise registry paths incorrectly
+# So we'll use the good old "reg" tool
+#Write-Host "##[command]dir -r ""HKLM:\Software\WOW6432Node\Python"""
+#dir -r "HKLM:\Software\WOW6432Node\Python"
+Write-Host "##[command]reg HKLM\Software\Python /s /reg:32"
+reg HKLM\Software\Python /s /reg:32
```

Comment:
Also, the containers don't seem to have `reg.exe`, so these steps probably need to be replaced with our own custom code for listing the registry.

Or maybe we can downgrade Powershell to a version that isn't broken by https://github.com/PowerShell/PowerShell/issues/5536